components/alert/AlertMap.vue

```diff
@@ -711,250 +867,181 @@ export default Vue.extend({
     },
     async loadRegions(): Promise<void> {
       if (this.mapRegions.size !== 0) {
-        return;
+        return
       }
 
-      const regions = await new MapRegionDataRequest()
-        .pull(this.alert.zone ? this.alert.zone : Zone.INDAR);
-      
-      this.$L.Icon.Default.prototype.options.iconAnchor = [0, 0];
-      this.$L.Icon.Default.prototype.options.tooltipAnchor = [0, 0];
+      const regions = await new MapRegionDataRequest().pull(
+        this.alert.zone ? this.alert.zone : Zone.INDAR
+      )
+
+      this.$L.Icon.Default.prototype.options.iconAnchor = [0, 0]
+      this.$L.Icon.Default.prototype.options.tooltipAnchor = [0, 0]
       regions.forEach((region) => {
-        this.mapRegions.set(region.id, region);
-        var polygon = this.$L.polygon(region.outline(), region.outlineOptions());
-        
+        this.mapRegions.set(region.id, region)
+        const polygon = this.$L.polygon(
+          region.outline(),
+          region.outlineOptions()
+        )
+
         // Set on hover colors and ensure the attached badge layers get the same event
-        polygon.on("mouseover", (e: L.LeafletMouseEvent) => {
+        polygon.on('mouseover', (e: L.LeafletMouseEvent) => {
           e.target.setStyle({
```

Comment:
This could be solved by a hover psudo class

Reply:
This exists so that the mouseover/mouseout events propagate to the facility badge and name even if just the facility hex is hovered over.

I'm not totally sure how you would propagate those events to the correct elements with just a hover pseudo class

Reply:
You're right, if it's event driven the hovers will do nothing!